config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -0,0 +1,137 @@
+      secret:
+        secretName: k8s-gcr-prod-service-account
+
+presubmits:
+  kubernetes/k8s.io:
+  - name: cip-dryrun
+    decorate: true
+    skip_report: false
+    run_if_changed: ".*/manifest.yaml"
+    # It's OK to run things concurrently, because we're only running things with -dry-run.
+    max_concurrency: 10
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: gcr.io/cip-demo-staging/cip@sha256:96be240bb072cb6b2b998e29daf87e1de4cd18c18d6ea97bec3f0cce487087ed
+        args:
+        - /bin/sh
+        - -c
+        - |
+          # TODO: Make the promoter manifest have per-repository service account
+          # credentials. That way, we can make this script just take a delta of
+          # "which manifest files were changed?" and only run the dry run
+          # against the ones that changed, instead of against all of them ilke
+          # this.
+          multirun.sh \
+            /app/cip-docker-image.binary \
+            k8s-staging-cluster-api/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json \
+            k8s-staging-coredns/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json \
+            k8s-staging-csi/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json
+        env:
+        - name: CIP_OPTS
+          # Just perform a dry run.
+          value: "-dry-run"
+        volumeMounts:
+        - name: k8s-gcr-prod-service-account-creds
+          mountPath: /etc/k8s-gcr-prod-service-account
+          readOnly: true
+      volumes:
+      - name: k8s-gcr-prod-service-account-creds
+        secret:
+          secretName: k8s-gcr-prod-service-account
+
+postsubmits:
+  cip-bot/cip-manifest:
+  - name: cip-prodrun
+    decorate: true
+    run_if_changed: "k8s.gcr.io/k8s-staging-cluster-api/manifest.yaml"
+    skip_report: false
+    # Never run more than 1 job at a time. This is because we don't want to run
+    # into a case where an older manifest PR merge gets run last (after a newer
+    # one).
+    max_concurrency: 1
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: gcr.io/cip-demo-staging/cip@sha256:96be240bb072cb6b2b998e29daf87e1de4cd18c18d6ea97bec3f0cce487087ed
+        args:
+        - /bin/sh
+        - -c
+        - |
+          multirun.sh \
+            /app/cip-docker-image.binary \
+            k8s-staging-cluster-api/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json \
+            k8s-staging-coredns/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json \
+            k8s-staging-csi/manifest.yaml,/etc/k8s-gcr-prod-service-account/service-account.json
+        imagePullPolicy: Always
+        volumeMounts:
+        - name: k8s-gcr-prod-service-account-creds
+          mountPath: /etc/k8s-gcr-prod-service-account
+          readOnly: true
+      volumes:
+      - name: k8s-gcr-prod-service-account-creds
+        secret:
+          secretName: k8s-gcr-prod-service-account
```

Comment:
Can you write a presubmit to validate that only `container-image-promoter.yaml` uses the `k8s-gcr-prod-service-account` secret?

See https://github.com/kubernetes/test-infra/blob/914cddb88731823139411680717f681eb6bd6778/config/tests/jobs/jobs_test.go#L311 for a similar check